lib/debugInfo.ts

```diff
@@ -1,8 +1,12 @@
-// @ts-nocheck
-const { version } = require('../package.json');
-const { getPlatform } = require('./environment');
+import { version } from '../package.json';
+import { getPlatform } from './environment';
+import { Arguments } from 'yargs';
 
-const logDebugInfo = ({ debug }) => {
+interface CustomOptions {
```

Comment:
If we're going to be passing these into `Arguments` a lot, which it seems like we are, probably makes sense to give them their own unique names. `DebugInfoFlagOptions`? `DebugInfoOptions`? Not sure, but CustomOptions doesn't tell us a lot and will also make searching for a specific `CustomOptions` type more difficult

Reply:
Just changed this to `Arguments<{ debug?: boolean }>` based on our Huddle on Friday afternoon. Going to resolve but feel free to reopen if need be.